tests/Formatter/CurlFormatterTest.php

```diff
@@ -132,4 +132,31 @@ public function testProperBodyReading()
         $this->assertContains("-d 'foo=bar&hello=world'", $curl);
         $this->assertContains("-X PUT", $curl);
     }
+
+    /**
+     * @expectedException \Symfony\Component\Debug\Exception\FatalErrorException
+     *
+     * @dataProvider getHeadersAndBodyData
+     */
+    public function testextractBodyArgument($headers, $body)
```

Comment:
@tchoulom totally my bad, dunno why I couldn't see that part in the github review.
Thanks, but the build is still failing